### PR TITLE
Add first search path setting command param.

### PR DESCRIPTION
**Pull request by chengstory**

### CODE REVIEW

**Comment by chengstory** [comment]
[ci rebuild]

File: tools/simulator/libsimulator/lib/ProjectConfig/ProjectConfig.cpp [hunk 1]
```diff
@@ -421,18 +409,11 @@ void ProjectConfig::parseCommandLine(const vector<string> &args)
             vector<string> pathes = split((*it), ';');
```
Comment (Naruto) [inline tools/simulator/libsimulator/lib/ProjectConfig/ProjectConfig.cpp:409]: I guess that delimiter character is ':' better than ';'.
';' is same as command delimiter for unix-shell.

File: tools/simulator/libsimulator/lib/ProjectConfig/ProjectConfig.h [hunk 2]
```diff
@@ -28,7 +28,7 @@ using namespace std;
 #define kProjectConfigDebugger                  1024    // -debugger-ldt, -debugger-codeide, -disable-debugger
 #define kProjectConfigListen                    2048    //
 #define kProjectConfigSearchPath                4096    //
-#define kProjectConfigUseLocalScript            8192    // -use-local-script
+#define kProjectConfigFirstSearchPath            8192    // -first-search-path
```
Comment (Naruto) [inline tools/simulator/libsimulator/lib/ProjectConfig/ProjectConfig.h:31]: add `kProjectConfigFirstSearchPath`  to `kProjectConfigAll`.

**Comment by chengstory** [comment]
[ci rebuild]

File: tools/simulator/libsimulator/lib/ProjectConfig/ProjectConfig.h (hunk 2 above)
Comment (chengstory) [inline tools/simulator/libsimulator/lib/ProjectConfig/ProjectConfig.h:31]: @Naruto  ok

**Comment by xiaofeng11** [comment]
[ci rebuild]

**Comment by xiaofeng11** [comment]
[ci rebuild]

**Comment by xiaofeng11** [comment]
[ci rebuild]

**Comment by zilongshanren** [comment]
[ci rebuild]

**Comment by xiaofeng11** [comment]
[ci rebuild]

**Pull request closed without merging**